### PR TITLE
UIP-47: Adding Jupyter notebook assets

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

* Added the Jupyter notebook css file and its dependencies
* Reorganised existing fonts into directories for each font
* Updated the scss/css files accordingly
* Added notes about updating the notebook css file when the narrative notebook version is updated

# Jira Ticket / Issue #

Related Jira ticket: https://kbase-jira.atlassian.net/browse/UIP-47
- [x] Added the Jira Ticket to the title of the PR (e.g. `UIP-007 Adds a PR template`)

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# Updating Version and Release Notes (if applicable)

- [ ] [Version has been bumped](https://semver.org/) for each release
- [ ] [Release notes](/RELEASE_NOTES.md) have been updated for each release (and during the merge of feature branches)

### CODE REVIEW

File: scss/partials/_landingPages.scss [hunk 1]
```diff
@@ -203,62 +203,6 @@ table {
   white-space: nowrap;
 }
 
-/* custom checkboxes (prototyped) */
```
Comment (ialarmedalien) [inline scss/partials/_landingPages.scss:206]: none of these are used anywhere

File: README.md [hunk 2]
```diff
@@ -30,6 +45,16 @@ npm run server_down
...
+
+Whenever the notebook is updated in the narrative repo, the accompanying style file should be updated in this repo. The easiest way to get the updated style file is to run the narrative locally using `kbase-narrative` and grab it from `http://localhost:8888/static/style/style.min.css`. The new `style.min.css` file should be added under [`assets/css`](assets/css).
+
+The Jupyter notebook css [relies upon fonts](https://github.com/jupyter/nbclassic/blob/main/bower.json) from the `bootstrap` and `font-awesome` npm packages, which are specified in the `package.json` file in this repo. The font files from those packages are copied to [`assets/components/`](assets/components/) to mimic the structure that the notebook css file expects. The node task `copy_fonts` (`npm run copy_fonts`) can be used to update the font files in the unlikely event that these versions of the fonts change.
```
Comment (briehl) [inline README.md:55]: 🎉

File: scss/partials/_landingPages.scss (hunk 1 above)
Comment (briehl) [reply scss/partials/_landingPages.scss:206]: Huh. That's funny. Maybe something older from kbase-ui?

**Review by briehl — APPROVED** [review]
Looks good!

**Pull request merged**